### PR TITLE
Add additional space before font-family if needed

### PR DESCRIPTION
**Pull request by smelukov**
`.8em"YS Text", Arial, Helvetica, sans-serif` => `.8em YS Text, Arial, Helvetica, sans-serif`

See #1519

### CODE REVIEW

**Comment by smelukov** [comment]
friendly ping ☺️

**Review by SedykhArtem — APPROVED** [review]

File: packages/postcss-minify-font-values/src/lib/minify-font.js [hunk 1]
```diff
@@ -56,9 +56,31 @@ module.exports = function (nodes, opts) {
...
 
-  familyStart += 2;
+  let needAdditionalSpace = false;
 
-  family = minifyFamily(nodes.slice(familyStart), opts);
+  // handle case .8em"Times New Roman"
+  if (nodes[familyStart + 1].type === 'space') {
+    familyStart += 2;
+  } else {
+    // @ts-ignore
```
Comment (ludofischer) [inline packages/postcss-minify-font-values/src/lib/minify-font.js:65]: I don't think we use `ts-ignore` anywhere else. Could you find a way to solve this through an additional check or a TypeScript cast?

**Review by ludofischer — CHANGES_REQUESTED** [review]
Thank you for your PR and sorry for taking so long to review it. It looks fine but could you solve the TypeScript issue?

File: packages/postcss-minify-font-values/src/lib/minify-font.js (hunk 1 above)
Comment (ludofischer) [reply packages/postcss-minify-font-values/src/lib/minify-font.js:65]: The tests do not pass either, there's likely too many assumptions about the shape of the nodes here.

**Pull request closed without merging**